Formula/freetype.rb

```diff
@@ -34,6 +34,10 @@ def install
     system "./configure", "--prefix=#{prefix}", "--without-harfbuzz"
     system "make"
     system "make", "install"
+
+    chdir prefix do
```

Comment:
@DomT4 👍 That does look better. https://github.com/Homebrew/homebrew-core/pull/4635/commits/a0255ce4e2add097d97bf9caf281c1aa3451dd18